roles/openshift_health_checker/openshift_checks/docker_image_availability.py

```diff
@@ -151,13 +160,12 @@ def available_images(self, images, registries, task_vars):
         ]
 
     def is_available_skopeo_image(self, image, registry, task_vars):
-        """Uses Skopeo to determine if required image exists in a given registry."""
+        """Uses Skopeo to determine if required image exists with a given registry."""
 
-        cmd_str = "skopeo inspect docker://{registry}/{image}".format(
-            registry=registry,
-            image=image,
-        )
+        # prepend registry only if image does not already have one
+        if image.count("/") < 2:
+            image = registry + "/" + image
```

Comment:
If the image is supposed to come from a specific registry (as with the etcd image or if they've put a registry in `oreg_url`) then it's not going to be accurate to look for it from just any registry... even if it's there, it wouldn't be used.